content/04.study.md

```diff
@@ -424,6 +424,92 @@ summarized above also apply to interfacial contact prediction for protein
+In the context of PPIs — as in other domains — deep learning shows promise both for exceeding current predictive performance and for circumventing limitations from which other approaches suffer.
+
+One of the key difficulties in applying deep learning techniques to binding prediction is the task of representing peptide and protein sequences in a meaningful way.
+DeepPPI [@doi:10.1021/acs.jcim.7b00028] made PPI predictions from a set of sequence and composition protein descriptors using a two-stage deep neural network.
+Sun et al. [@doi:10.1186/s12859-017-1700-2] applied autocovariances, a coding scheme which returns uniform-size vectors describing the covariance between variables at various positions.
+A stacked autoencoder paired with a softmax classifier then gave predictions as probabilities of interaction.
+Wang et al. [@doi:10.1039/C7MB00188F] used deep learning as an intermediate step in PPI prediction.
+They first extracted sequence descriptors called Zernike moments, which have often been used in image analysis.
+A stacked sparse autoencoder was then used to reduce feature dimensions and noisiness before a novel type of classification vector machine made PPI predictions.
+
+Beyond predicting whether or not two proteins interact, Du et al. [@doi:10.1016/j.ymeth.2016.06.001] showed that a tandem stacked-autoencoder/deep-neural-network method could be used to predict residue contacts for the interfacial regions of interacting proteins.
+A combination of a hidden Markov model with Fisher scores yielded uniform-length features for each residue. Their method significantly exceeded classical machine learning accuracy.
+
+Because many studies used predefined higher-level features, one of the benefits of deep learning — automatic feature extraction — is not fully leveraged.
+More work is needed to determine the best ways to represent raw protein sequence information so that the full benefits of deep learning as an automatic feature extractor can be realized.
+
+#### MHC-peptide binding
+
+An important type of PPI involves the immune system's ability to recognize the body's own cells.
+The major histocompatibility complex (MHC) plays a key role in regulating this process by binding antigens and displaying them on the cell surface to be recognized by T cells.
+Due to its importance in immunity and immune response, peptide-MHC binding prediction is an important problem in computational biology.
+Pan-allelic MHC-peptide binding prediction is a useful area of research as well, due to the highly polymorphic nature of the MHC-encoding gene region.
+Several approaches have shown promise towards both of these tasks.
+
+Shallow, feed-forward neural networks are competitive methods and have made progress toward pan-allele and pan-length peptide prediction.
```

Comment:
Instead of listing each method and a few of its properties, is there a way to condense this paragraph and highlight some of the most important differences between approaches?  I'm not sure what those are exactly.  Maybe things like variable versus fixed length input.  Some of the pre-2010 papers must have had some limitations that have been improved on in the deep learning era.